packages/thirdweb/src/extensions/marketplace/english-auctions/write/bidInAuction.ts

```diff
@@ -106,3 +107,19 @@ export function bidInAuction(
     },
   });
 }
+
+/**
+ * Checks if the `bidInAuction` method is supported by the given contract.
+ * @param availableSelectors An array of 4byte function selectors of the contract. You can get this in various ways, such as using "whatsabi" or if you have the ABI of the contract available you can use it to generate the selectors.
+ * @returns A boolean indicating if the `bidInAuction` method is supported.
+ * @extension MARKETPLACE
+ * @example
+ * ```ts
+ * import { isBidInAuctionSupported } from "thirdweb/extensions/marketplace";
+ *
+ * const supported = isBidInAuctionSupported(["0x..."]);
+ * ```
+ */
+export function isBidInAuctionSupported(availableSelectors: string[]) {
+  return BidInAuction.isBidInAuctionSupported(availableSelectors);
+}
```

Comment:
missing functions, this needs to support every function that is used in the actual execution